server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileCopier.java

```diff
@@ -75,7 +75,7 @@ private QualityProfileDto prepareTarget(String fromProfileKey, String toName) {
       if (toProfile == null) {
         // Do not delegate creation to QProfileBackuper because we need to keep
         // the parent-child association, if exists.
-        toProfile = factory.create(dbSession, qProfileWsSupport.getDefaultOrganization(dbSession), toProfileName);
+        toProfile = factory.checkAndCreate(dbSession, qProfileWsSupport.getDefaultOrganization(dbSession), toProfileName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getDefaultOrganization"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)